netpuppy/__main__.py

```diff
@@ -7,6 +7,12 @@
 import socket
 import sys
 
+def network_port(value):
+    ivalue = int(value)
+    if ivalue <= 0 or ivalue > 65535:
```

Comment:
Also not super critical but maybe more pythonic you can simplify the if logic using chained comparison

```python
if not 0 <= value <= 65535:
```